weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallOverview.tsx

```diff
@@ -26,63 +42,15 @@ export const CallOverview: React.FC<{
   }, [call.opVersionRef]);
   const callOp = useOpVersion(opVersionKey);
   const opCategory = callOp.result?.category;
-  const childCalls = useCalls(call.entity, call.project, {
-    parentIds: [call.callId],
-  });
-  const safeChildCalls = useMemo(() => {
-    if (childCalls.loading) {
-      return [];
-    }
-    return (childCalls.result ?? []).filter(c => c.opVersionRef != null);
-  }, [childCalls.loading, childCalls.result]);
-  const attributes = _.fromPairs(
-    Object.entries(span.attributes ?? {}).filter(
-      ([k, a]) => !k.startsWith('_') && a != null
-    )
-  );
-  const summary = _.fromPairs(
-    Object.entries(span.summary ?? {}).filter(
-      ([k, a]) => !k.startsWith('_') && k !== 'latency_s' && a != null
-    )
-  );
+
+  const statusCode = call.rawSpan.status_code;
 
   return (
-    <SimpleKeyValueTable
-      data={{
-        Operation:
-          parseRefMaybe(span.name) != null ? (
-            <SmallRef objRef={parseRefMaybe(span.name)!} wfTable="OpVersion" />
-          ) : (
-            span.name
-          ),
-        ...(opCategory
-          ? {
-              Category: <CategoryChip value={opCategory} />,
-            }
-          : {}),
-        Status: <StatusChip value={span.status_code} />,
-        Called: <Timestamp value={span.timestamp / 1000} format="relative" />,
-        ...(span.summary.latency_s != null
-          ? {
-              Latency: span.summary.latency_s.toFixed(3) + 's',
-            }
-          : {}),
-        ...(span.exception ? {Exception: span.exception} : {}),
-        ...((safeChildCalls.length ?? 0) > 0
-          ? {
-              'Child Calls': (
-                <GroupedCalls
-                  calls={safeChildCalls}
-                  partialFilter={{
-                    parentId: call.callId,
-                  }}
-                />
-              ),
-            }
-          : {}),
-        ...(Object.keys(attributes).length > 0 ? {Attributes: attributes} : {}),
-        ...(Object.keys(summary).length > 0 ? {Summary: summary} : {}),
-      }}
-    />
+    <Overview>
```

Comment:
This section feels pretty empty now and seems to take up more space than needed with the large padding. Also, it visually looks redundant with the main header. I wonder if we could just remove this overview section and bring these details into the header?

Reply:
I don't like the redundancy either, let's discuss it with design.